README.md

```diff
@@ -7,3 +8,16 @@
 
 # Clinical Data Flow
 Project management and design artifacts for [Clinical Data Flow](https://d3b.io/docs/products/clinical-data-flow) product
+
+## Evolving Architecture
+[Draw.io Source](https://drive.google.com/file/d/1xdtljwZ2FjFVxJD4tPRbP1QWk2j1eE3d/view?usp=sharing)
```

Comment:
Add KF dataservice into diagram

Reply:
In the long run, we also want to persist records from REDCap into the CDF warehouse, right?